install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -377,7 +377,7 @@ spec:
         - sh
         - -c
         - |
-          until test -f {{ .Values.nodeinit.bootstrapFile | quote }}; do
+          until test -s {{ .Values.nodeinit.bootstrapFile | quote }}; do
```

Comment:
do you know which part node-init script writes to this file /tmp/cilium-bootstrap-time ? I can't find in either [startup.bash](https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/files/nodeinit/startup.bash) or [manage-startup-script.sh](https://github.com/cilium/image-tools/blob/master/images/startup-script/manage-startup-script.sh)

Reply:
This one https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/files/nodeinit/startup.bash#L121